generateproto.go

```diff
@@ -17,4 +17,5 @@
 package perun
 
 //go:generate protoc --proto_path=proto --go_out=api/grpc/pb --go-grpc_out=api/grpc/pb proto/nodetypes.proto proto/errors.proto proto/payment_service.proto
-//go:generate protoc --proto_path=proto --go_out=api/grpc/pb --go-grpc_out=api/grpc/pb proto/sdktypes.proto proto/funding_service.proto
+//go:generate protoc --proto_path=proto --go_out=api/grpc/pb --go-grpc_out=api/grpc/pb proto/sdktypes.proto proto/watching_service.proto
+//go:generate protoc --proto_path=proto --go_out=api/grpc/pb --go-grpc_out=api/grpc/pb proto/sdktypes.proto proto/watching_service.proto
```

Comment:
why is this entry doubled?

Reply:
I made a error while copying. Renamed both lines to watching_service, instead of just the copy

Reply:
Fixed in this force push: https://github.com/hyperledger-labs/perun-node/compare/b2a5067768ca2b1dc216ca42bbfbcba223f895a2..d5dcd874064ea43bb0f4192b23ff3f39f8665c6c